distribution/docs/src/main/resources/_contents/_catalog-contents/integrating-catalog-contents.adoc

```diff
@@ -1730,7 +1730,8 @@ The `validator` key must have a value of one of the following:
  * `arguments`: (unlimited) [list of strings: each argument is one case-sensitive, valid enumeration value]
   - `enumerationignorecase`
   * `arguments`: (unlimited) [list of strings: each argument is one case-insensitive, valid enumeration value]
-
+ - `iso3_country` (validates that a String is a valid ISO 3166-1 alpha-3 country code)
```

Comment:
It appears we have an ignoreCase validator but it is not installed by default or documented.